gulpfile.mjs

```diff
@@ -2184,13 +2183,13 @@ gulp.task(
       console.log();
       console.log("### Cloning baseline distribution");
 
-      rimraf.sync(DIST_DIR);
-      mkdirp.sync(DIST_DIR);
+      fs.rmSync(DIST_DIR, { recursive: true, force: true });
+      fs.mkdirSync(DIST_DIR, { recursive: true });
       safeSpawnSync("git", ["clone", "--depth", "1", DIST_REPO_URL, DIST_DIR]);
 
       console.log();
       console.log("### Overwriting all files");
-      rimraf.sync(path.join(DIST_DIR, "*"));
+      fs.rmSync(DIST_DIR, { recursive: true, force: true });
```

Comment:
That's the only line I'm not sure of

Reply:
If you run `gulp dist-pre` locally and compare the result with/without this patch you'll see that this change breaks things, since the output is no longer identical.

Hence this patch would *reintroduce* issue #15975, note in particular https://github.com/mozilla/pdf.js/issues/15975#issuecomment-1445400924.

Reply:
Hmmm, trying something else. I don't see the difference in outputs now, but I need a fresh brain to be sure.